spec/javascripts/components/story/expanded_story/expanded_story_description/expanded_story_description_content_area_spec.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ExpandedStoryContentArea from 'components/story/ExpandedStory/ExpandedStoryDescription/ExpandedStoryContentArea';
+
+describe('<ExpandedStoryContentArea />', () => {
+  const renderContent = propsOverride =>
+    shallow (
+      <ExpandedStoryContentArea
+        onClick={sinon.stub()}
+        {...propsOverride}
+      />
+    )
+
+  describe("when description isn't null", () => {
+    const description = 'description example';
+
+    it('renders <ExpandedStoryDescriptionContent />', () => {
+      const wrapper = renderContent({ description });
+      const descriptionContent = wrapper.find('[data-id="description-content"]');
+      expect(descriptionContent.exists()).toBeTruthy();
+    });
+
+    it('does not render <ExpandedStoryDescriptionEditButton />', () => {
+      const wrapper = renderContent({ description });
+      const editButton = wrapper.find('[data-id="edit-button"]');
+      expect(editButton.exists()).toBeFalsy();
+    });
+  });
+
+  const invalidsDescriptions = [null, undefined, ''];
+  invalidsDescriptions.forEach(description => {
+    describe(`when description is ${description}`, () => {
+      let wrapper;
+      let descriptionContent;
+      let editButton;
+  
+      beforeEach(() => {
+        wrapper = renderContent({ description });
```

Comment:
Let's avoid rendering in `beforeEach`. If you need to repeat the part you search for the description and the button, make `renderContent` return them as keys, something like this:

```js
const { editButton } = renderContent({ description });
```

But it should be done inside the `it`.